webapp/security/views.py

```diff
@@ -263,8 +269,32 @@ def api_create_notice():
 # CVE views
 # ===
 def cve_index():
-    return flask.render_template("security/cve/index.html")
+
+    # Temporary fix for data to be displayed
+    cves_query = db_session.query(CVE).first()
+    if not cves_query and os.environ["FLASK_DEBUG"] == "true":
+        create_data()
+    list_cve = db_session.query(CVE).limit(10)
+
+    return flask.render_template("security/cve/index.html", list_cve=list_cve)
 
 
 def cve(cve_id):
-    return flask.render_template("security/cve/cve.html")
+
+    # Temporary fix for data to be displayed
+    cves_query = db_session.query(CVE).first()
+    if not cves_query and os.environ["FLASK_DEBUG"] == "true":
```

Comment:
Can we please remove this? I suspect this will actually fail the second time you load the page, since it will attempt to insert a CVE with an ID that already exists. It is acceptable to just execute the helper script before you start developing anything CVE related. IMO we should not be loading data at runtime.